src/components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/SelectExistingPlatform.tsx

```diff
@@ -48,7 +46,8 @@ const SelectExistingPlatform = ({ onClose }) => {
             "application/vnd.google-apps.form"
 
           const isAddButtonDisabled =
```

Comment:
Yes, we could use the new `usePlatformUsageInfo` hook here, but this solution is actually simpler in this specific case, so I left it as is.